common/services/conversion/track.ts

```diff
@@ -126,24 +126,34 @@ function trackSegmentEvent({
 
 function track(conversion: Conversion) {
   const debug = getCookie(cookies.analyticsDebug) === true;
+  const toggles = Object.entries(getCookies())
+    .map(([k, v]) => {
+      const isToggleCookie = k.match(/toggle_/);
+
+      return isToggleCookie && { [k]: v === 'true' };
+    })
+    .filter(Boolean);
```

Comment:
By sheer coincidence, I've implemented similar logic in #8957. It might be worth us pulling that out into a shared function.

Reply:
[this is nicer](https://github.com/wellcomecollection/wellcomecollection.org/pull/8957/files#diff-845354ee43ad39c46c43eb2922bc711766589a7711e4ab152c64430a7efc35c2R63-R66) than what I've got – I'll split it out and use that. Thanks!